test/coverage.ts

```diff
@@ -27,126 +27,126 @@ export interface TestRunnerOptions {
 
 export class CoverageRunner {
 
```

Comment:
@sudhirverma Coverage fix should be in a different PR. It does not relate to this PR fix.